core/dispatcher/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -493,13 +493,13 @@ class ContainerPool(
         val network = config.invokerContainerNetwork
         val policy = config.invokerContainerPolicy
         val env = getContainerEnvironment()
-        val pull = !imageName.contains("whisk/")
+        val pull = !imageName.contains(config.dockerImagePrefix + "/")
```

Comment:
action is not in scope here and can't easily put in scope, as this method is used to run stemcell containers without an action being set.